AttributeError in fetching javascript for admin panel

This seems to only happen when you start the server using `start` and then try to visit `/admin/jsi18n/`:

```
[2017-07-12 16:04:40 -0500] [1423] [ERROR] Internal Server Error: /admin/jsi18n/
Traceback (most recent call last):
  File "/Users/jathan/sandbox/virtualenvs/nsot/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/jathan/sandbox/virtualenvs/nsot/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 254, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
  File "/Users/jathan/sandbox/virtualenvs/nsot/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/jathan/sandbox/virtualenvs/nsot/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 233, in inner
    return view(request, *args, **kwargs)
  File "/Users/jathan/sandbox/virtualenvs/nsot/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 346, in i18n_javascript
    return javascript_catalog(request, packages=['django.conf', 'django.contrib.admin'])
  File "/Users/jathan/sandbox/virtualenvs/nsot/lib/python2.7/site-packages/django/views/i18n.py", line 304, in javascript_catalog
    locale = to_locale(get_language())
  File "/Users/jathan/sandbox/virtualenvs/nsot/lib/python2.7/site-packages/django/utils/translation/__init__.py", line 194, in to_locale
    return _trans.to_locale(language)
  File "/Users/jathan/sandbox/virtualenvs/nsot/lib/python2.7/site-packages/django/utils/translation/trans_real.py", line 75, in to_locale
    p = language.find('-')
AttributeError: 'NoneType' object has no attribute 'find'
[2017-07-12 16:04:40 -0500] [1423] [INFO] 500 GET /admin/jsi18n/ (127.0.0.1) 354.66ms
```

Something is screwy which is preventing admin panel widgets from properly loading, displaying only their basic styled HTML rendering. This can be seen in the group add/edit panel where you're selecting permissions for a group.

--- a/nsot/admin.py
+++ b/nsot/admin.py
@@ -13,7 +13,7 @@ from . import models
 class UserAdmin(EmailUserAdmin):
     fieldsets = (
         (None, {
-            'fields': ('email', 'secret_key'),
+            'fields': ('email', 'secret_key', 'password'),
         }),
         (_('Permissions'), {
             'fields': (

--- a/nsot/util/commands.py
+++ b/nsot/util/commands.py
@@ -15,6 +15,11 @@ __all__ = ('NsotCommand', 'CommandError')
 
 class NsotCommand(BaseCommand):
     """Base management command for NSoT that implements a custom logger."""
+
+    # This is here to alleviate an AttributeError when getting /admin/jsi18n/
+    # Ref: https://github.com/dropbox/nsot/issues/279
+    leave_locale_alone = True
+
     def create_parser(self, prog_name, subcommand):
         """Override default parser to include default values in help."""
         parser = super(NsotCommand, self).create_parser(

